extensions/mqtt/processors/PublishMQTT.h

```diff
@@ -137,7 +137,7 @@ class PublishMQTT : public processors::AbstractMQTTProcessor {
   std::shared_ptr<logging::Logger> logger_;
 };
 
-REGISTER_RESOURCE(PublishMQTT);
+REGISTER_RESOURCE(PublishMQTT, "This Processor puts the contents of a FlowFile to a MQTT broker for a specified topic. The content of a FlowFile becomes the payload of the MQTT message.");
```

Comment:
```
contents of a FlowFile
```
content?

Reply:
In plural form contents can referenced in a concrete sense to things contained within something. This isn't incorrect. I'll err on avoiding to change the original author's wording.

Reply:
Naturally I'm not a native speaker, the reason I asked is having both plural and singular usage of content when it's about a single flow file.

Reply:
Hmm that's a good point. I'll see if I can clarify this, then.